providers/software_sources.rb

```diff
@@ -18,7 +18,15 @@
       
       current_lists = []
       remote_lists = []    
-      remote_lists.push("gecosv2.list")
+      
+      case $gecos_os
+      when "GECOS V3"
+        remote_lists.push("official-package-repositories.list")
```

Comment:
Ah! Ok. I checked the dependencies with "apt-cache rdepends mintsources" and saw that "mint-meta-core".
The "gecosws-cinnamon-conf" dependency was updated by me to configure mintsources properly. Mintsources is the replacement of "software-properties-gtk". That is an application that in GECOS V2 was in the menu as "Software and updates" (see atached image).

But I think that we could remote it. Let me make the necessary changes.
![software_and_updates](https://cloud.githubusercontent.com/assets/4385829/20568871/34096ae8-b19e-11e6-850f-ee091efa26fb.png)